docusaurus/docs/Android/03-ui/04-components/03-message-list.md

```diff
@@ -121,23 +228,181 @@ TransformStyle.messageListStyleTransformer = StyleTransformer { defaultViewStyle
 }
 ```
 
+## Channel features flags
 
-## Creating a Custom Empty State
+Some xml attributes provide possibility to enable/disable some features. For example:
+1. R.attrs.streamUiReplyEnabled - defines if users can reply to messages
+2. R.attrs.streamUiCopyMessageActionEnabled - defines if users can copy messages
+3. R.attrs.streamUiEditMessageEnabled - defines if users can edit their messages
+4. R.attrs.streamUiMuteUserEnabled - defines if users can mute others
+5. R.attrs.streamUiDeleteMessageEnabled - defines if users can delete their messages
+6. Others you can find the full list of attributes [here](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/attrs_message_list_view.xml)
```

Comment:
Do we have other flags? Maybe in that case it's worth it to enumerate all of them?